packages/playground/blueprints/src/lib/steps/enable-multisite.ts

```diff
@@ -34,6 +30,8 @@ export interface EnableMultisiteStep {
 export const enableMultisite: StepHandler<EnableMultisiteStep> = async (
 	playground
 ) => {
+	await installWpCli(playground, {});
```

Comment:
This will allow JS API users to execute commands even if they didn't include the `extraLibrary`.